src/module/classes/barbarian.mjs

```
@@ -1,32 +1,31 @@
  */
-export function feral(actor,combatants){
-    let improved = true
-    if (improved){
-        improvedFeral(actor,combatants)
-    }else{
-        
-        // console.log("Nothing ")
+export function feral(actor) {
+    const rageFeature = actor.items.find(i => i.name === "Rage");
+    if (rageFeature && actor.items.find(i => i.name === "Feral Instincts") && !actor.items.find(i => i.name === "Improved Feral Instincts")) {
+        let uses = rageFeature.system.uses.max - rageFeature.system.uses.spent;
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'uses' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)